### DISCUSSION TITLE
Help installing via Docker Compose

### DISCUSSION THREAD

**Discussion by youboutna** [inventree/InvenTree#6292] [Q&A] [answered]
Hello,
I am trying to install InevnTree via Docker Compose.
------ commands  that I executed : ----

1)  docker-compose build --no-cache 
ok . 
2) docker-compose up 

crash with   errors 

Traceback (most recent call last):
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 255, in fetch_command
inventree-inventree-dev-server-1  |     app_name = commands[subcommand]
inventree-inventree-dev-server-1  | KeyError: 'wait_for_db'
inventree-inventree-dev-server-1  | 
inventree-inventree-dev-server-1  | During handling of the above exception, another exception occurred:
inventree-inventree-dev-server-1  | 
inventree-inventree-dev-server-1  | Traceback (most recent call last):
inventree-inventree-dev-server-1  |   File "/home/inventree/InvenTree/manage.py", line 24, in <module>
inventree-inventree-dev-server-1  | 
inventree-inventree-dev-server-1  |     main()
inventree-inventree-dev-server-1  |   File "/home/inventree/InvenTree/manage.py", line 20, in main
inventree-inventree-dev-server-1  |     execute_from_command_line(sys.argv)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
inventree-inventree-dev-server-1  |     utility.execute()
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 436, in execute
inventree-inventree-dev-server-1  |     self.fetch_command(subcommand).run_from_argv(self.argv)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 262, in fetch_command
inventree-inventree-dev-server-1  |     settings.INSTALLED_APPS
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 89, in __getattr__
inventree-inventree-dev-server-1  |     self._setup(name)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 76, in _setup
inventree-inventree-dev-server-1  |     self._wrapped = Settings(settings_module)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 190, in __init__
inventree-inventree-dev-server-1  |     mod = importlib.import_module(self.SETTINGS_MODULE)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
inventree-inventree-dev-server-1  |     return _bootstrap._gcd_import(name[level:], package, level)
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap_external>", line 883, in exec_module
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
inventree-inventree-dev-server-1  |   File "/home/inventree/InvenTree/InvenTree/settings.py", line 24, in <module>
inventree-inventree-dev-server-1  |     import moneyed
inventree-inventree-dev-server-1  | ModuleNotFoundError: No module named 'moneyed'
inventree-inventree-dev-worker-1  | Traceback (most recent call last):
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 255, in fetch_command
inventree-inventree-dev-worker-1  |     app_name = commands[subcommand]
inventree-inventree-dev-worker-1  | KeyError: 'wait_for_db'
inventree-inventree-dev-worker-1  | 
inventree-inventree-dev-worker-1  | During handling of the above exception, another exception occurred:
inventree-inventree-dev-worker-1  | 
inventree-inventree-dev-worker-1  | Traceback (most recent call last):
inventree-inventree-dev-worker-1  |   File "/home/inventree/InvenTree/manage.py", line 24, in <module>
inventree-inventree-dev-worker-1  |     main()
inventree-inventree-dev-worker-1  |   File "/home/inventree/InvenTree/manage.py", line 20, in main
inventree-inventree-dev-worker-1  | 
inventree-inventree-dev-worker-1  |     execute_from_command_line(sys.argv)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
inventree-inventree-dev-worker-1  |     utility.execute()
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 436, in execute
inventree-inventree-dev-worker-1  |     self.fetch_command(subcommand).run_from_argv(self.argv)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 262, in fetch_command
inventree-inventree-dev-worker-1  |     settings.INSTALLED_APPS
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 89, in __getattr__
inventree-inventree-dev-worker-1  |     self._setup(name)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 76, in _setup
inventree-inventree-dev-worker-1  |     self._wrapped = Settings(settings_module)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 190, in __init__
inventree-inventree-dev-worker-1  |     mod = importlib.import_module(self.SETTINGS_MODULE)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
inventree-inventree-dev-worker-1  |     return _bootstrap._gcd_import(name[level:], package, level)
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap_external>", line 883, in exec_module
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
inventree-inventree-dev-worker-1  |   File "/home/inventree/InvenTree/InvenTree/settings.py", line 24, in <module>
inventree-inventree-dev-worker-1  |     import moneyed
inventree-inventree-dev-worker-1  | ModuleNotFoundError: No module named 'moneyed'
inventree-inventree-dev-server-1 exited with code 0
inventree-inventree-dev-worker-1 exited with code 0
inventree-inventree-dev-server-1  | Traceback (most recent call last):
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 255, in fetch_command
inventree-inventree-dev-server-1  |     app_name = commands[subcommand]
inventree-inventree-dev-server-1  | KeyError: 'wait_for_db'
inventree-inventree-dev-server-1  | 
inventree-inventree-dev-server-1  | During handling of the above exception, another exception occurred:
inventree-inventree-dev-server-1  | 
inventree-inventree-dev-server-1  | Traceback (most recent call last):
inventree-inventree-dev-server-1  |   File "/home/inventree/InvenTree/manage.py", line 24, in <module>
inventree-inventree-dev-server-1  | 
inventree-inventree-dev-server-1  |     main()
inventree-inventree-dev-server-1  |   File "/home/inventree/InvenTree/manage.py", line 20, in main
inventree-inventree-dev-server-1  |     execute_from_command_line(sys.argv)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
inventree-inventree-dev-server-1  |     utility.execute()
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 436, in execute
inventree-inventree-dev-server-1  |     self.fetch_command(subcommand).run_from_argv(self.argv)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 262, in fetch_command
inventree-inventree-dev-server-1  |     settings.INSTALLED_APPS
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 89, in __getattr__
inventree-inventree-dev-server-1  |     self._setup(name)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 76, in _setup
inventree-inventree-dev-server-1  |     self._wrapped = Settings(settings_module)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 190, in __init__
inventree-inventree-dev-server-1  |     mod = importlib.import_module(self.SETTINGS_MODULE)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
inventree-inventree-dev-server-1  |     return _bootstrap._gcd_import(name[level:], package, level)
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap_external>", line 883, in exec_module
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
inventree-inventree-dev-server-1  |   File "/home/inventree/InvenTree/InvenTree/settings.py", line 24, in <module>
inventree-inventree-dev-server-1  |     import moneyed
inventree-inventree-dev-server-1  | ModuleNotFoundError: No module named 'moneyed'
inventree-inventree-dev-worker-1  | Traceback (most recent call last):
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 255, in fetch_command
inventree-inventree-dev-worker-1  |     app_name = commands[subcommand]
inventree-inventree-dev-worker-1  | KeyError: 'wait_for_db'
inventree-inventree-dev-worker-1  | 
inventree-inventree-dev-worker-1  | During handling of the above exception, another exception occurred:
inventree-inventree-dev-worker-1  | 
inventree-inventree-dev-worker-1  | Traceback (most recent call last):
inventree-inventree-dev-worker-1  |   File "/home/inventree/InvenTree/manage.py", line 24, in <module>
inventree-inventree-dev-worker-1  | 
inventree-inventree-dev-worker-1  |     main()
inventree-inventree-dev-worker-1  |   File "/home/inventree/InvenTree/manage.py", line 20, in main
inventree-inventree-dev-worker-1  |     execute_from_command_line(sys.argv)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
inventree-inventree-dev-worker-1  |     utility.execute()
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 436, in execute
inventree-inventree-dev-worker-1  |     self.fetch_command(subcommand).run_from_argv(self.argv)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 262, in fetch_command
inventree-inventree-dev-worker-1  |     settings.INSTALLED_APPS
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 89, in __getattr__
inventree-inventree-dev-worker-1  |     self._setup(name)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 76, in _setup
inventree-inventree-dev-worker-1  |     self._wrapped = Settings(settings_module)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 190, in __init__
inventree-inventree-dev-worker-1  |     mod = importlib.import_module(self.SETTINGS_MODULE)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
inventree-inventree-dev-worker-1  |     return _bootstrap._gcd_import(name[level:], package, level)
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap_external>", line 883, in exec_module
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
inventree-inventree-dev-worker-1  |   File "/home/inventree/InvenTree/InvenTree/settings.py", line 24, in <module>
inventree-inventree-dev-worker-1  |     import moneyed
inventree-inventree-dev-worker-1  | ModuleNotFoundError: No module named 'moneyed'
inventree-inventree-dev-server-1 exited with code 0
inventree-inventree-dev-worker-1 exited with code 0
inventree-inventree-dev-server-1  | Traceback (most recent call last):
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 255, in fetch_command
inventree-inventree-dev-server-1  |     app_name = commands[subcommand]
inventree-inventree-dev-server-1  | KeyError: 'wait_for_db'
inventree-inventree-dev-server-1  | 
inventree-inventree-dev-server-1  | During handling of the above exception, another exception occurred:
inventree-inventree-dev-server-1  | 
inventree-inventree-dev-server-1  | Traceback (most recent call last):
inventree-inventree-dev-server-1  |   File "/home/inventree/InvenTree/manage.py", line 24, in <module>
inventree-inventree-dev-server-1  | 
inventree-inventree-dev-server-1  |     main()
inventree-inventree-dev-server-1  |   File "/home/inventree/InvenTree/manage.py", line 20, in main
inventree-inventree-dev-server-1  |     execute_from_command_line(sys.argv)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
inventree-inventree-dev-server-1  |     utility.execute()
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 436, in execute
inventree-inventree-dev-server-1  |     self.fetch_command(subcommand).run_from_argv(self.argv)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 262, in fetch_command
inventree-inventree-dev-server-1  |     settings.INSTALLED_APPS
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 89, in __getattr__
inventree-inventree-dev-server-1  |     self._setup(name)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 76, in _setup
inventree-inventree-dev-server-1  |     self._wrapped = Settings(settings_module)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 190, in __init__
inventree-inventree-dev-server-1  |     mod = importlib.import_module(self.SETTINGS_MODULE)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
inventree-inventree-dev-server-1  |     return _bootstrap._gcd_import(name[level:], package, level)
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap_external>", line 883, in exec_module
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
inventree-inventree-dev-server-1  |   File "/home/inventree/InvenTree/InvenTree/settings.py", line 24, in <module>
inventree-inventree-dev-server-1  |     import moneyed
inventree-inventree-dev-server-1  | ModuleNotFoundError: No module named 'moneyed'
inventree-inventree-dev-worker-1  | Traceback (most recent call last):
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 255, in fetch_command
inventree-inventree-dev-worker-1  |     app_name = commands[subcommand]
inventree-inventree-dev-worker-1  | KeyError: 'wait_for_db'
inventree-inventree-dev-worker-1  | 
inventree-inventree-dev-worker-1  | During handling of the above exception, another exception occurred:
inventree-inventree-dev-worker-1  | 
inventree-inventree-dev-worker-1  | Traceback (most recent call last):
inventree-inventree-dev-worker-1  |   File "/home/inventree/InvenTree/manage.py", line 24, in <module>
inventree-inventree-dev-worker-1  | 
inventree-inventree-dev-worker-1  |     main()
inventree-inventree-dev-worker-1  |   File "/home/inventree/InvenTree/manage.py", line 20, in main
inventree-inventree-dev-worker-1  |     execute_from_command_line(sys.argv)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
inventree-inventree-dev-worker-1  |     utility.execute()
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 436, in execute
inventree-inventree-dev-worker-1  |     self.fetch_command(subcommand).run_from_argv(self.argv)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 262, in fetch_command
inventree-inventree-dev-worker-1  |     settings.INSTALLED_APPS
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 89, in __getattr__
inventree-inventree-dev-worker-1  |     self._setup(name)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 76, in _setup
inventree-inventree-dev-worker-1  |     self._wrapped = Settings(settings_module)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 190, in __init__
inventree-inventree-dev-worker-1  |     mod = importlib.import_module(self.SETTINGS_MODULE)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
inventree-inventree-dev-worker-1  |     return _bootstrap._gcd_import(name[level:], package, level)
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap_external>", line 883, in exec_module
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
inventree-inventree-dev-worker-1  |   File "/home/inventree/InvenTree/InvenTree/settings.py", line 24, in <module>
inventree-inventree-dev-worker-1  |     import moneyed
inventree-inventree-dev-worker-1  | ModuleNotFoundError: No module named 'moneyed'
inventree-inventree-dev-server-1 exited with code 1
inventree-inventree-dev-worker-1 exited with code 1
inventree-inventree-dev-server-1  | Traceback (most recent call last):
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 255, in fetch_command
inventree-inventree-dev-server-1  |     app_name = commands[subcommand]
inventree-inventree-dev-server-1  | KeyError: 'wait_for_db'
inventree-inventree-dev-server-1  | 
inventree-inventree-dev-server-1  | During handling of the above exception, another exception occurred:
inventree-inventree-dev-server-1  | 
inventree-inventree-dev-server-1  | Traceback (most recent call last):
inventree-inventree-dev-server-1  |   File "/home/inventree/InvenTree/manage.py", line 24, in <module>
inventree-inventree-dev-server-1  |     main()
inventree-inventree-dev-server-1  |   File "/home/inventree/InvenTree/manage.py", line 20, in main
inventree-inventree-dev-server-1  |     execute_from_command_line(sys.argv)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
inventree-inventree-dev-server-1  |     utility.execute()
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 436, in execute
inventree-inventree-dev-server-1  |     self.fetch_command(subcommand).run_from_argv(self.argv)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 262, in fetch_command
inventree-inventree-dev-server-1  |     settings.INSTALLED_APPS
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 89, in __getattr__
inventree-inventree-dev-server-1  |     self._setup(name)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 76, in _setup
inventree-inventree-dev-server-1  |     self._wrapped = Settings(settings_module)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 190, in __init__
inventree-inventree-dev-server-1  |     mod = importlib.import_module(self.SETTINGS_MODULE)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
inventree-inventree-dev-server-1  |     return _bootstrap._gcd_import(name[level:], package, level)
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap_external>", line 883, in exec_module
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
inventree-inventree-dev-server-1  |   File "/home/inventree/InvenTree/InvenTree/settings.py", line 24, in <module>
inventree-inventree-dev-server-1  |     import moneyed
inventree-inventree-dev-server-1  | ModuleNotFoundError: No module named 'moneyed'
inventree-inventree-dev-server-1  | 
inventree-inventree-dev-worker-1  | Traceback (most recent call last):
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 255, in fetch_command
inventree-inventree-dev-worker-1  |     app_name = commands[subcommand]
inventree-inventree-dev-worker-1  | KeyError: 'wait_for_db'
inventree-inventree-dev-worker-1  | 
inventree-inventree-dev-worker-1  | During handling of the above exception, another exception occurred:
inventree-inventree-dev-worker-1  | 
inventree-inventree-dev-worker-1  | Traceback (most recent call last):
inventree-inventree-dev-worker-1  |   File "/home/inventree/InvenTree/manage.py", line 24, in <module>
inventree-inventree-dev-worker-1  |     main()
inventree-inventree-dev-worker-1  |   File "/home/inventree/InvenTree/manage.py", line 20, in main
inventree-inventree-dev-worker-1  | 
inventree-inventree-dev-worker-1  |     execute_from_command_line(sys.argv)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
inventree-inventree-dev-worker-1  |     utility.execute()
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 436, in execute
inventree-inventree-dev-worker-1  |     self.fetch_command(subcommand).run_from_argv(self.argv)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 262, in fetch_command
inventree-inventree-dev-worker-1  |     settings.INSTALLED_APPS
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 89, in __getattr__
inventree-inventree-dev-worker-1  |     self._setup(name)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 76, in _setup
inventree-inventree-dev-worker-1  |     self._wrapped = Settings(settings_module)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 190, in __init__
inventree-inventree-dev-worker-1  |     mod = importlib.import_module(self.SETTINGS_MODULE)
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
inventree-inventree-dev-worker-1  |     return _bootstrap._gcd_import(name[level:], package, level)
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap_external>", line 883, in exec_module
inventree-inventree-dev-worker-1  |   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
inventree-inventree-dev-worker-1  |   File "/home/inventree/InvenTree/InvenTree/settings.py", line 24, in <module>
inventree-inventree-dev-worker-1  |     import moneyed
inventree-inventree-dev-worker-1  | ModuleNotFoundError: No module named 'moneyed'
inventree-inventree-dev-server-1 exited with code 1
inventree-inventree-dev-worker-1 exited with code 1
inventree-inventree-dev-server-1  | Traceback (most recent call last):
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 255, in fetch_command
inventree-inventree-dev-server-1  |     app_name = commands[subcommand]
inventree-inventree-dev-server-1  | KeyError: 'wait_for_db'
inventree-inventree-dev-server-1  | 
inventree-inventree-dev-server-1  | During handling of the above exception, another exception occurred:
inventree-inventree-dev-server-1  | 
inventree-inventree-dev-server-1  | Traceback (most recent call last):
inventree-inventree-dev-server-1  |   File "/home/inventree/InvenTree/manage.py", line 24, in <module>
inventree-inventree-dev-server-1  | 
inventree-inventree-dev-server-1  |     main()
inventree-inventree-dev-server-1  |   File "/home/inventree/InvenTree/manage.py", line 20, in main
inventree-inventree-dev-server-1  |     execute_from_command_line(sys.argv)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
inventree-inventree-dev-server-1  |     utility.execute()
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 436, in execute
inventree-inventree-dev-server-1  |     self.fetch_command(subcommand).run_from_argv(self.argv)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 262, in fetch_command
inventree-inventree-dev-server-1  |     settings.INSTALLED_APPS
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 89, in __getattr__
inventree-inventree-dev-server-1  |     self._setup(name)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 76, in _setup
inventree-inventree-dev-server-1  |     self._wrapped = Settings(settings_module)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/site-packages/django/conf/__init__.py", line 190, in __init__
inventree-inventree-dev-server-1  |     mod = importlib.import_module(self.SETTINGS_MODULE)
inventree-inventree-dev-server-1  |   File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
inventree-inventree-dev-server-1  |     return _bootstrap._gcd_import(name[level:], package, level)
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap_external>", line 883, in exec_module
inventree-inventree-dev-server-1  |   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
inventree-inventree-dev-server-1  |   File "/home/inventree/InvenTree/InvenTree/settings.py", line 24, in <module>
inventree-inventree-dev-server-1  |     import moneyed
inventree-inventree-dev-server-1  | ModuleNotFoundError: No module named 'moneyed'
inventree-inventree-dev-worker-1  | Traceback (most recent call last):
inventree-inventree-dev-worker-1  |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 255, in fetch_command
inventree-inventree-dev-worker-1  |     app_name = commands[subcommand]
inventree-inventree-dev-worker-1  | KeyError: 'wait_for_db'
inventree-inventree-dev-worker-1  | 
inventree-inventree-dev-worker-1  | During handling of the above exception, another exception occurred:
inventree-inventree-dev-worker-1  | 
inventree-inventree-dev-worker-1  | Traceback (most recent call last):
inventree-inventree-dev-worker-1  |   File "/home/inventree/InvenTree/manage.py", line 24, in <module>

**Comment by SchrodingersGat** [comment, marked as answer]
You need to run `docker compose run inventree-dev-server invoke update` before `docker compose up`

As you are building the development image, you should be following these instructions here:

https://docs.inventree.org/en/stable/start/docker_dev/#quickstart-guide

However, are you intending on building the development image? Or are you intending to use for production?

If development, I'd highly recommend following the [devcontainer process](https://docs.inventree.org/en/stable/develop/devcontainer/) to get a local dev environment going.

If production, use the [docker production guide](https://docs.inventree.org/en/stable/start/docker_prod/)

**Reply by youboutna** [reply]
Thank you @SchrodingersGat, I executed command docker compose run inventree-dev-server invoke update. For now I use it in Dev ENV.
